src/test/java/uk/gov/hmcts/reform/sscs/service/OutOfHoursCalculatorTest.java

```diff
@@ -55,7 +55,7 @@ public void getStartOfNextInHoursPeriodWhenItIsTheNextDay() throws NoSuchAlgorit
         assertThat(nextInHoursTime.getMonthValue(), is(9));
         assertThat(nextInHoursTime.getDayOfMonth(), is(19));
         assertThat(nextInHoursTime.getHour(), is(START_HOUR));
-        assertThat(new Double(nextInHoursTime.getMinute()), closeTo(0, 59));
+        assertThat(Double.valueOf(nextInHoursTime.getMinute()), closeTo(0, 59));
```

Comment:
I appreciate the effort... however it was bad before and it still is. I would suggest either to leave it alone or fixing its core problem.

`closeTo` in this case effectively means that any value between -59 and 59 will be accepted. Of course that's not what's intended here. What was intended was to validate that the value is between 0 and 59.

If you really want to fix it, try this:
`assertThat(nextInHoursTime.getMinute(), in(IntStream.range(0, 59).boxed().collect(Collectors.toSet())));`

Reply:
I changed it to a assertj assertion